src/main/java/net/dv8tion/jda/client/entities/impl/JDAClientImpl.java

```diff
@@ -347,4 +348,32 @@ protected void handleResponse(Response response, Request<AuthorizedApplication>
             }
         };
     }
+
+	@Override
+	public RestAction<Group> createGroupDM(ArrayList<User> users)
+	{
+		if(users.size() < 2)
+			throw new IllegalArgumentException("Users array must contain at least two users!");
+		if(users.size() > 9)
+			throw new IllegalArgumentException("Group DM cannot contain more than 10 members! (You are the tenth member, max array size is 9)");
+		
+		JSONArray array = new JSONArray();
+		
+		for(User user : users){
+			array.put(user.getId());
+		}
+		
+		Route.CompiledRoute route = Route.Self.CREATE_PRIVATE_CHANNEL_V6.compile();
+        return new RestAction<Group>(api, route, new JSONObject().put("recipients", array))
+        {
+            @Override
+            protected void handleResponse(Response response, Request<Group> request)
+            {
+                if (response.isOk()){
+                    request.onSuccess(api.getEntityBuilder().createGroup(response.getObject()));
+                }else
```

Comment:
all brackets must be on new line, if you use a bracket for `if` you have to use one on `else` as well. The `else` must be on its own respective line